### ISSUE TITLE
Error after npm run dev

### ISSUE THREAD

**Issue by sugarsyy** [Thulinma/ACNLPatternTool#9]
```
ℹ ｢wdm｣: wait until bundle finished: /
Could not resolve "/ACNLPatternTool/zxing-js-library/esm5" in file /ACNLPatternTool/src/components/FileLoader.vue.
Could not resolve "/ACNLPatternTool/zxing-js-library/esm5" in file /ACNLPatternTool/src/libs/ACNLQRGenerator.js.
ℹ ｢wdm｣: Failed to compile.
```

I have installed the project dependencies but I get the above error message after starting the project. Can you help? Thanks~

**Comment by DamSenViet** [comment]
Please read the README on the dev branch. I left instructions there to properly setup everything! Hope that helps.

**Comment by ryanmcgrath** [comment]
Worth noting that I ran into the same issue, and had to go in and manually `npm build` in `zxing-js-library` to get the project to build/run. There was some other tinkering too, but I forget offhand what it was.

Point is... nothing with Webpack is ever simple and I'm not sure the repo docs accurately depict how to build/run, lol

**Comment by DamSenViet** [comment]
At the moment, the master branch readme doesn't reflect how to setup. The command structure does partially rely on knowledge of how node uses args. E.g. npm run build -- <options> command requires the -- to properly pass args and everything after can be parsed like typical Unix commands. An example command would be something like npm run build -- --help. It's more of an npm limitation, but I believe I documented enough on the development branch. That's why I added npm run build:submodule. Hope that helps!
